Dockerfile

```diff
@@ -71,6 +71,7 @@ RUN mkdir -p db
 COPY --from=builder --chown=wgui:wgui /build/wg-ui .
 RUN chmod +x wg-ui
 COPY init.sh .
+RUN chmod +x init.sh
```

Comment:
Why do we need this? The `init.sh` file is configured to be executed by any user already.

Reply:
ah ok, that makes sense. thank you for your explanation